cmd/push.go

```diff
@@ -52,11 +52,14 @@ type pushOptions struct {
 func Push(ctx context.Context) *cobra.Command {
 	pushOpts := &pushOptions{}
 	cmd := &cobra.Command{
-		Use:   "push [svc]",
-		Short: "Build, push and redeploy source code to the target app",
-		Args:  utils.MaximumNArgsAccepted(1, "https://okteto.com/docs/reference/cli/#push"),
+		Hidden: true,
+		Use:    "push [svc]",
```

Comment:
does `okteto push` without arguments still work if using an old manifest?

Reply:
yes, It will get the old manifest and use that as the dev manifest